ts/Core/Legend.ts

```diff
@@ -1857,6 +1757,30 @@ class Legend {
 
 }
 
+/* *
+ *
+ *  Class Prototype
+ *
+ * */
+
+interface Legend extends LegendLike {
+    // use declare module pattern to add
+}
+
+/* *
+ *
+ *  Class Namespace
+ *
+ * */
+
+namespace Legend {
+    export interface LegendCheckBoxElement extends HTMLDOMElement {
+        checked?: boolean;
```

Comment:
The interface name should be reduced to `CheckBoxElement` as it is in the `Legend` namespace.